apis/swagger.yml

```diff
@@ -351,6 +351,30 @@ paths:
           schema:
            $ref: "#/definitions/Error"
       tags: ["Container"]
+  /containers/{id}/pause:
+    post:
+      summary: "Pause a container"
+      operationId: "ContainerPause"
+      parameters:
+        - name : "id"
+          in : "path"
+          required: true
+          description: "ID or name of the container"
+          type: "string"
+      responses:
+        204:
+          description: "no error"
+        404:
+          description: "no such container"
```

Comment:
We should add `#definitions/Error` here as well, I think.
If other places missed this, maybe we could add that in follow-up pull requests.